download.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Define repository details
+USER="aaron-medina-aligent"         # Replace with the GitHub username or organization
+REPO="test-shop-repo"       # Replace with the repository name
+BRANCH="main"           # Replace with the branch name, e.g., "main" or "master"
+
+# GitHub API URL to fetch all files in the repository
+API_URL="https://api.github.com/repos/$USER/$REPO/git/trees/$BRANCH?recursive=1"
```

Comment:
this looks like it's just downloading files from a repository. should we be using `git clone`?

Reply:
The content of this repository is intended for setting up new client Shopify repositories, and we aim to keep it free of unnecessary Git metadata such as the .git directory, branch details, commit history, and tags. To achieve this, I’ve created a script that downloads only the files. This approach avoids tracking information and eliminates the extra step of decompressing ZIP files.

Reply:
wouldn't we be able to still use `git clone namespace/package --depth=1 && rm .git`?

Reply:
I’ve added this option to the README file. I’m also considering further enhancing the scripts.